### PR TITLE
Improve deployment scripts, make service account JSON configurable by directory, not name

### PR DESCRIPTION
**Pull request by davidz627**
Fixes: #28

Added some comments to the deployment scripts. JSON is finally configurable by directory, not name.

/assign @msau42

### CODE REVIEW

File: deploy/setup-project.sh [hunk 1]
```diff
@@ -1,13 +1,26 @@
 #!/bin/bash
 
+# This script will setup the given project with a Service Account that has the correct
+# restricted permissions to run the gcp_compute_persistent_disk_csi_driver and download
+# the keys to a specified directory
+
+# WARNING: This script will delete and recreate all service accounts, bindings, and keys.
```
Comment (msau42) [inline deploy/setup-project.sh]: ALL service accounts???

File: deploy/setup-project.sh [hunk 2]
```diff
@@ -1,13 +1,26 @@
...
+# Great care must be taken to not run the script with a service account that is currently
+# in use.
+
+# Args:
+# PROJECT: GCP project
+# GCE_PD_SA_NAME: Name of the service account to create
+# GCE_PD_SA_DIR: Directory to save the service account key
```
Comment (msau42) [inline deploy/setup-project.sh]: Are there defaults set?

File: deploy/setup-project.sh (hunk 1 above)
Comment (davidz627) [reply deploy/setup-project.sh]: ohoh thats alarming. No just the one set.

File: deploy/setup-project.sh (hunk 2 above)
Comment (davidz627) [reply deploy/setup-project.sh]: Should there be defaults? I think these are all important enough that there shouldn't be any defaults. I have `set -o nounset` so it throws an error if the vars are unset when the script runs

File: deploy/kubernetes/deploy-driver.sh [hunk 3]
```diff
@@ -1,5 +1,15 @@
 #!/bin/bash
 
+# This script will deploy the GCP Compute Persistent Disk CSI Driver to the
+# currently available Kubernetes cluster
+
+# Note: setup-cluster.yaml depends on the existence of cluster-roles
+# system:csi-external-attacher and system:csi-external-provisioner
+# which are in Kubernetes version 1.10.5+
```
Comment (msau42) [inline deploy/kubernetes/deploy-driver.sh]: I would put this note in the readme right up top

**Comment by davidz627** [comment]
Comments addressed

**Pull request merged**